content/en/llm_observability/instrumentation/auto_instrumentation.md

```diff
@@ -297,6 +298,20 @@ The MCP integration instruments the following methods:
   - `mcp.server.fastmcp.tools.tool_manager.ToolManager.call_tool`
 
 
+## Google ADK
+
+The Google ADK integration automatically agent runs, tool calls, and code executions made through [Google's ADK Python SDK][60].
```

Comment:
```suggestion
The Google ADK integration automatically performs Agent runs, tool calls, and code executions made through [Google's ADK Python SDK][60].
```
I think this is missing a word here, correct me if I'm wrong !

Reply:
@git-thuerk-done "Agent" shouldn't be capitalized here. It's in reference to AI agents, not the Datadog Agent. But yes does look like there's a missing word.

Reply:
Thank you @cswatt ! Let me adjust my review :)

Reply:
oh yes a word is missing 😭 thanks for catching!!

Reply:
updated!